contracts/adapters/Onboarding.sol

```diff
@@ -42,6 +42,10 @@ contract OnboardingContract is
 {
     using SafeMath for uint256;
 
+    bytes32 constant ChunkSize = keccak256("chunkSize");
```

Comment:
to avoid ambiguous properties, I would prefix them with some context (usually the adapter) unless it is self explanatory

so here I would call it "onboarding.chunkSize" and "onboarding.sharesPerChunk" etc ... WDYT ?

Reply:
Makes sense.